content/faq/downgrades.md

```diff
@@ -10,7 +10,7 @@ Since STARTTLS negotiation is done in the open, the process of upgrading an emai
 
 #### DANE
 
-DANE, or [DNS-based Authentication of Named Entities](https://tools.ietf.org/html/rfc7672) relies on DNSSEC, a protocol for publishing and authenticating signed DNS entries. Mailserver operators would sign their MX records with DNSSEC, and publish a record alongside them containing the public key that the mailserver is expected to use in TLS. So long as senders trust the DNSSEC signature chain, senders can discover both 1) whether to expect a STARTTLS connection, and 2) how to validate the recipient's certificate. Full DANE deployment presents a scalable solution for mailservers to clarify certificate validation rules and prevent downgrade attacks. However, DANE is dependent on deployment and validation of DNSSEC, and which has has unfortunately [remained stagnant](https://www.internetsociety.org/deploy360/dnssec/statistics/) (validation is at around 10-15% worldwide) for the past five years.
+DANE, or [DNS-based Authentication of Named Entities](https://tools.ietf.org/html/rfc7672) relies on DNSSEC, a protocol for publishing and authenticating signed DNS entries. Mailserver operators would sign their MX records with DNSSEC and publish a record alongside them containing the public key that the mailserver is expected to use in TLS. So long as senders trust the DNSSEC signature chain, senders can discover both 1) whether to expect a STARTTLS connection, and 2) how to validate the recipient's certificate. Full DANE deployment presents a scalable solution for mailservers to clarify certificate validation rules and prevent downgrade attacks. However, DANE is dependent on deployment and validation of DNSSEC, which has unfortunately [remained stagnant](https://www.internetsociety.org/deploy360/dnssec/statistics/) (validation is at around 10-15% worldwide) for the past five years.
 
 The Internet Society has a [good explanation and list of related resources](https://www.internetsociety.org/resources/deploy360/dane/) if you're interested in deploying both DNSSEC and DANE.
 
```

Comment:
"It's in last call at the IETF, so few MTA software have implemented it so far." sounds like it might be a typo to me, but I'm not sure how to correct it.

Reply:
Maybe "few MTAs"?